### PR TITLE
Marauder Limit Fix

### PR DESCRIPTION
**Pull request by Robustin**
Jesus that scaling proc was a clusterfuck. Recent_Marauders was not being updated at all.

### CODE REVIEW

File: code/modules/antagonists/clockcult/clock_scriptures/scripture_applications.dm [hunk 1]
```diff
@@ -80,15 +80,13 @@
 	usage_tip = "Reciting this scripture multiple times in a short period will cause it to take longer!"
 	tier = SCRIPTURE_APPLICATION
 	one_per_tile = TRUE
-	primary_component = BELLIGERENT_EYE
```
Comment (Xhuis) [inline code/modules/antagonists/clockcult/clock_scriptures/scripture_applications.dm]: ?

File: code/modules/antagonists/clockcult/clock_scriptures/scripture_applications.dm [hunk 2]
```diff
@@ -98,32 +96,20 @@
...
-	var/WT = world.time
-	var/MT = time_since_last_marauder //Cast it for quicker reference
-	var/marauders_to_exclude = 0
-	if(world.time >= time_since_last_marauder + MARAUDER_SCRIPTURE_SCALING_THRESHOLD)
-		marauders_to_exclude = round(WT - MT) / MARAUDER_SCRIPTURE_SCALING_THRESHOLD //If at least 20 seconds have passed, lose one marauder for each 20 seconds
-		//i.e. world.time = 10000, last marauder = 9000, so we lose 5 marauders from the recent count since 10k - 9k = 1k, 1k / 200 = 5
-		time_since_last_marauder = world.time //So that it can't be spammed to make the marauder exclusion plummet; this emulates "ticking"
-	recent_marauders = max(0, recent_marauders - marauders_to_exclude)
-	scaled_recital_time = min(recent_marauders * MARAUDER_SCRIPTURE_SCALING_TIME, MARAUDER_SCRIPTURE_SCALING_MAX)
+	sleep(MARAUDER_SCRIPTURE_SCALING_THRESHOLD)
```
Comment (Xhuis) [inline code/modules/antagonists/clockcult/clock_scriptures/scripture_applications.dm]: Sleeps are very bad. Use an addtimer to call a separate proc to decrement `recent_marauders`.

Comment (Robustin) [reply code/modules/antagonists/clockcult/clock_scriptures/scripture_applications.dm]: I did and it didn't work.

Probably the same reason the proc I removed didn't work either (aside from it having broken math).

Comment (Xhuis) [reply code/modules/antagonists/clockcult/clock_scriptures/scripture_applications.dm]: If adding a timer didn't work, it's generally not done properly. I'm too tired to write some pseudocode myself, though.

Comment (coiax) [reply code/modules/antagonists/clockcult/clock_scriptures/scripture_applications.dm]: Don't use sleep.

**Pull request closed without merging**